pkg/login/other_logins_check.go

```diff
@@ -0,0 +1,151 @@
+package login
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	logger "github.com/sirupsen/logrus"
+
+	authenticationapi "k8s.io/api/authentication/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	BACKPLANE_USER_NAMESPACE_PREFIX = "openshift-backplane-"
+	CEE                             = "cee"
+	MCS_TIER_TWO                    = "mcs-tier-two"
+	LPSRE                           = "lpsre"
+	SREP                            = "srep"
+)
+
+var backplane_user_namespaces_to_check []string = []string{
+	CEE,
+	MCS_TIER_TWO,
+	LPSRE,
+	SREP,
+}
+
+func CheckForOtherSessions(config *rest.Config) {
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		logger.Errorln("Unable to build kube client from rest config to check for additional sessions")
+	}
+
+	token, err := getTokenFromConfig(config)
+	if err != nil {
+		logger.Error("Unable to get token for self review to find other sessions")
+		return
+	}
+
+	me, err := whoami(clientset, token)
+	if err != nil {
+		logger.Error("Unable to determine who I am to find other sessions")
+		return
+	}
+	meSlice := strings.Split(me, ":")
+	// meSlice should always be something like
+	// system, serviceaccount, openshift-backplane-role-name, username
+	myNS := meSlice[2]
+	myUser := meSlice[3]
+	logger.Debugf("I am %s:%s", myNS, myUser)
+	sessions := map[string]int{}
+
+	for _, role := range backplane_user_namespaces_to_check {
```

Comment:
Based on the PR description, it indicates that after a successful login, the user (likely MCS?) should be notified if there are existing SRE backplane sessions. This helps MCS determine if an SRE is already paged and investigating an issue on the cluster? 

However, the current check loops through `cee` and `mcs-tier-two` as well. If the goal is to detect only SRE presence, should `mcs-tier-two` and `cee` be included in the check?

Reply:
The goal is for anyone who logs into a cluster with backplane to see any other users who have logged in with backplane who's users have not been cleaned up yet.

So everyone sees everyone, not any other one way around.